README.md

```diff
@@ -27,24 +29,39 @@ To use Voici, you'll need to install it first:
 pip install voici
 ```
 
-Then, you can generate static dashboards from a directory of Notebooks like this:
+Then, you can generate static dashboards from a notebook or a directory of Notebooks like this:
 
 ```bash
-voici build --contents notebooks/
+# Build a single dashboard
+voici my-notebook.ipynb
+# Build a directory of notebooks
+voici notebooks/
 ```
 
-Once your dashboards built, you can simply serve them with a simple static server, _e.g._:
+Once your dashboards are built, you can simply serve them with a simple static server, _e.g._:
 
 ```bash
 cd _output
 python -m http.server
 ```
 
-The `voici` command line interface is the same as the `jupyter lite` one. The only difference is that the `voici build` command will only generate Voici dashboards, excluding the full JupyterLab interface from the output. Running `voici build .` is equivalent to running `jupyter lite build . --apps voici`
+## Advanced usage
```

Comment:
Should these go to the docs? https://voici.readthedocs.io/en/latest/

And then maybe just point to the docs and remove the README instructions (or keep them simple) to avoid too much duplication?